server/scsynth/SC_ComPort.cpp

```diff
@@ -443,12 +443,10 @@ SC_TcpConnection::~SC_TcpConnection() { mParent->connectionDestroyed(); }
 //////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 static void asioFunction() {
+    /* NB: on macOS we just keep the default thread priority */
 #ifdef NOVA_TT_PRIORITY_RT
-    std::pair<int, int> priorities = nova::thread_priority_interval_rt();
-    nova::thread_set_priority_rt((priorities.first + priorities.second) / 2);
-#else
-    std::pair<int, int> priorities = nova::thread_priority_interval();
-    nova::thread_set_priority(priorities.second);
+    int priority = nova::thread_priority_interval_rt().first;
+    nova::thread_set_priority_rt(priority);
```

Comment:
Should this apply to all platforms?

Reply:
`NOVA_TT_PRIORITY_RT` is defined for Windows and Linux.

Reply:
So this change also changes behavior on Linux, is that correct?

Reply:
Yes, it slightly changes the value passed to `nova::thread_priority_interval_rt`, but as I mentioned in the PR description, this is in line with Supernova.

Reply:
Ah, so "in line with Supernova" also applies to Linux, thanks for clarifying.

For a good measure, I'd like to test the performance on Windows and Linux before pulling it in. (IMO it's all good on macOS)